packages/devtools-reps/src/object-inspector/utils.js

```diff
@@ -283,21 +386,30 @@ function createNode(name, path, contents) {
   // lists. The path will be separated by property name,
   // i.e. `{ foo: { bar: { baz: 5 }}}` will have a path of `foo/bar/baz`
   // for the inner object.
-  return { name, path, contents };
+  return { parent, name, path, contents };
 }
 
-function getChildren({ getObjectProperties, actors, item }) {
+function getChildren({
+  getObjectEntries,
+  getObjectProperties,
+  actors,
+  item
+}) {
```

Comment:
this is a minor point, but it would be really nice to add flow types to these utils